src/components/Contact.jsx

```diff
@@ -1,124 +1,117 @@
-import React from "react";
+import React, {useState} from "react";
 import contact_illustration from "../contact_illustration.svg";
 
-export default class MyForm extends React.Component {
-  constructor(props) {
-    super(props);
-    this.submitForm = this.submitForm.bind(this);
-    this.state = {
-      status: "",
-    };
-  }
+function Contact() {
+    const [status, setStatus] = useState("");
+
+    const submitForm = (ev) => {
+        ev.preventDefault();
+        const form = ev.target;
+        const data = new FormData(form);
+        const xhr = new XMLHttpRequest();
+        xhr.open(form.method, form.action);
+        xhr.setRequestHeader("Accept", "application/json");
+        xhr.onreadystatechange = () => {
+            if (xhr.readyState !== XMLHttpRequest.DONE) return;
+            if (xhr.status === 200) {
+                form.reset();
+                setStatus("SUCCESS");
+            } else {
+                setStatus("ERROR");
+            }
+        };
+        xhr.send(data);
+    }
```

Comment:
You can replace this with axios! We already have it in my dependencies.

Reply:
Ok, I'll do that.